tests/api_workflow/mocked_api_workflow_client.py

```diff
@@ -214,14 +372,81 @@ def delete_tag_by_tag_id(self, dataset_id, tag_id, **kwargs):
+                        "fileName": "2008_006249_jpg.rf.fdd64460945ca901aa3c7e48ffceea83.jpg",
+                        "exif": {},
+                        "index": 0,
+                        "createdAt": 1647852727873,
+                        "lastModifiedAt": 1647852727873,
+                        "metaData": {
+                            "sharpness": 27.31265790443818,
+                            "sizeInBytes": 48224,
+                            "snr": 2.1969673926211217,
+                            "mean": [
+                                0.24441662557257224,
+                                0.4460417517905863,
+                                0.6960984853824035,
+                            ],
+                            "shape": [167, 500, 3],
+                            "std": [
+                                0.12448681278605961,
+                                0.09509570033043004,
+                                0.0763725998175394,
+                            ],
+                            "sumOfSquares": [
+                                6282.243860049413,
+                                17367.702452895475,
+                                40947.22059208768,
+                            ],
+                            "sumOfValues": [
+                                20408.78823530978,
+                                37244.486274513954,
+                                58124.22352943069,
+                            ],
+                        },
+                    },
+                },
+            }
+        ]
 
+    def export_tag_to_label_box_data_rows(
+        self, dataset_id: str, tag_id: str, **kwargs
+    ) -> List[LabelBoxDataRow]:
+        if kwargs["page_offset"] and kwargs["page_offset"] > 0:
+            return []
+        return [
+            {
+                "externalId": "2008_007291_jpg.rf.2fca436925b52ea33cf897125a34a2fb.jpg",
+                "imageUrl": "https://api.lightly.ai/v1/datasets/62383ab8f9cb290cd83ab5f9/samples/62383cb7e6a0f29e3f31e233/readurlRedirect?type=CENSORED",
+            }
+        ]
 
-    def export_tag_to_label_box_data_rows(self, dataset_id: str, tag_id: str):
-        return [{'externalId': '2008_007291_jpg.rf.2fca436925b52ea33cf897125a34a2fb.jpg', 'imageUrl': 'https://api.lightly.ai/v1/datasets/62383ab8f9cb290cd83ab5f9/samples/62383cb7e6a0f29e3f31e233/readurlRedirect?type=CENSORED'}]
-    
-    def export_tag_to_basic_filenames(self, dataset_id: str, tag_id: str):
+    def export_tag_to_basic_filenames_and_read_urls(
+        self, dataset_id: str, tag_id: str, **kwargs
+    ) -> List[FilenameAndReadUrl]:
+        if kwargs["page_offset"] and kwargs["page_offset"] > 0:
+            return []
+        return [
+            {
+                "fileName": "export-basic-test-sample-0.png",
+                "readUrl": "https://storage.googleapis.com/somwhere/export-basic-test-sample-0.png?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=CENSORED",
+            }
```

Comment:
Please don't return a `List[Dict]`, but a `List[FilenameAndReadUrl]` instead

Reply:
can you please share code? I don't follow what you mean